Source/Immutable/Private/Immutable/Actions/ImtblPassportConnectEvmAsyncAction.cpp

```diff
@@ -0,0 +1,54 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "Immutable/Actions/ImtblPassportConnectEvmAsyncAction.h"
+
+#include "Immutable/ImmutablePassport.h"
+#include "Immutable/ImmutableSubsystem.h"
+#include "Immutable/Misc/ImtblLogging.h"
+
+
+UImtblPassportConnectEvmAsyncAction* UImtblPassportConnectEvmAsyncAction::ConnectEvm(UObject* WorldContextObject)
+{
+    UImtblPassportConnectEvmAsyncAction* PassportInitBlueprintNode = NewObject<UImtblPassportConnectEvmAsyncAction>();
+    PassportInitBlueprintNode->WorldContextObject = WorldContextObject;
+    return PassportInitBlueprintNode;
+}
+
+
+void UImtblPassportConnectEvmAsyncAction::Activate()
+{
+    if (!WorldContextObject || !WorldContextObject->GetWorld())
+    {
+        FString Err = "ConnectEvm failed due to missing world or world context object.";
+        IMTBL_WARN("%s", *Err)
```

Comment:
```suggestion
        IMTBL_WARN("ConnectEvm failed due to missing world or world context object.")
```
Should be able to just use the string, no need to reference it. Same for the other action files